config/webpack.config.js

```diff
@@ -210,7 +216,7 @@ module.exports = function (env, isomorphic = false, noAnimation = false, ilibAdd
 			// We don't currently advertise code splitting but Webpack supports it.
 			filename: '[name].js',
 			// There are also additional JS chunk files if you use code splitting.
-			chunkFilename: 'chunk.[name].js',
+			chunkFilename: contenthash ? 'chunk.[name].[contenthash].js' : 'chunk.[name].js',
```

Comment:
Why did you add this only to chunk files?
I think we could use it to `filename` also.
How about assets like image files?

Reply:
For several reasons, the filename also needs to be changed.(image file, apply main's changes, etc.)
I will change this.

Reply:
Where is the change for "assets" like image files?

Reply:
I just add it too.
(verification result: check WRP-3673)